wlcreate/Start-Asian-Love
# Resolves 1. As a user, I want to see a image carousel on the Home page

## Description



Issue number: closes #1 

This PR adds `react slick` (and `slick carousel` for css) so that we may use the package as our Home page's carousel. We chose this package after narrowing it down to three choices because of the documentation and features `react slick` provides. 

Resources:
- [React Slick npm package](https://www.npmjs.com/package/react-slick)
- [React Slick GitHub repo](https://github.com/akiran/react-slick)
- [React Slick docs](https://react-slick.neostack.com/docs/example/simple-slider)
- [Styling with NextJS](https://www.smashingmagazine.com/2020/09/comparison-styling-methods-next-js/)
- [Specifying ports for development](https://onecompiler.com/posts/3w5u9patd/how-to-change-port-in-next-js-app)

## Acceptance Criteria

- finding the right package
- figuring how many images we want (4-6)
- placeholders for what we want

## Self-check

Please check all that apply:


## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓  | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

<img width="1505" alt="Screen Shot 2021-04-07 at 7 04 42 PM" src="https://user-images.githubusercontent.com/62153993/113945506-6c411e80-97d4-11eb-9f9d-4d04445e4190.png">

### After

[![Image from Gyazo](https://i.gyazo.com/4240a900a768353938f5a9e0a1b670e9.gif)](https://gyazo.com/4240a900a768353938f5a9e0a1b670e9)

## Testing Steps / QA Criteria


- switch to this feature branch: `git checkout image-carousel`
- get the new dependences: `npm install`
- start the app with `npm run dev` 
- check to see if the carousel appears and if you can browse through the slides
